src/components/FieldText.js

```diff
@@ -16,6 +16,21 @@ const mapError = ({
     }
     : { ...input, ...props });
 
+const required = value => (value ? undefined : 'Required');
+
+const email = value =>
```

Comment:
Looks good enough for our purposes.

The truth is, email addresses have a horribly complex format. Just look at [these examples](https://en.wikipedia.org/wiki/Email_address#Examples).

Someone made a ['correct'](http://www.ex-parrot.com/~pdw/Mail-RFC822-Address.html) regexp that parses email addresses. It just shows you how bad it is. (And even this one fails to do the length check)

In practice, no-one cares about the official standard and instead they use a subset like you do.  I've never seen anyone use the commenting mechanism or the double quotes.